code_schemes/s01mag16.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Please can we change to Q5 everywhere we're using 'quarter'? Dates are generally in Q1, Q2 etc.